### PR TITLE
Fix maps sample

### PR DESCRIPTION
**Pull request by tishan89**
## Purpose
Fix maps sample

## Approach
casting to string has become a safe operation. Hence changed the code to reflect that.

### CODE REVIEW

File: docs/ballerina-by-example/examples/fork-join/fork-join.bal [hunk 1]
```diff
@@ -27,7 +27,7 @@ function main (string[] args) {
         // Get values received from worker 'w1'.
         any[] resW1 =? <any[]>results["w1"];
         int iW1 =? <int>resW1[0];
-        string sW1 =? <string>resW1[1];
+        string sW1 = <string>resW1[1];
```
Comment (afkham) [inline docs/ballerina-by-example/examples/fork-join/fork-join.bal]: Casting to string has become a safe operation by design or by accident?

Comment (tishan89) [reply docs/ballerina-by-example/examples/fork-join/fork-join.bal]: Was under the impression it was by design. @lankavitharana can you please confirm.

Comment (tishan89) [reply docs/ballerina-by-example/examples/fork-join/fork-join.bal]: I check against the latest build and behavior is still the same

Comment (hasithaa) [reply docs/ballerina-by-example/examples/fork-join/fork-join.bal]: @afkham It is by design. This was discussed in https://groups.google.com/d/topic/ballerina-dev/sQkpK7iW32g/discussion

**Comment by tishan89** [comment]
Opened PR[1]. Hence closing this.

[1] https://github.com/ballerina-lang/ballerina/pull/6027

**Pull request closed without merging**